### DISCUSSION TITLE
[2.3.2] ATT&CK Navigator default playbook is blank

### DISCUSSION THREAD

**Discussion by BjornA77** [Security-Onion-Solutions/securityonion#1786] [General] [answered]
When I installed version 2.3.1 just for testing in a VM I got the Navigator populated with a lot of fields in default playbook.
Now when I have made a new install of 2.3.2 on a physical machine the Navigator is blank as shown in the picture I attached.
I have tried both Eval and Standalone install and installed all components.

Is it supposed to be blank as default or am I missing something? Even if I go to the Playbook link in SO and activate a new playbook the Navigator is still blank.

The documentation for SO 2.3 says:
"The default layer is titled Playbook and is automatically updated when a Play from Playbook is made active/inactive. This allows you to see your Detection Playbook coverage across the ATT&CK framework."

![image](https://user-images.githubusercontent.com/73762058/97916144-dd2a6900-1d52-11eb-91b7-015906cb0370.png)

**Comment by weslambert** [comment, marked as answer]
I've noticed the same issue, created the following for tracking: https://github.com/Security-Onion-Solutions/securityonion/issues/1798
